build/gen_stub.php

```diff
@@ -1155,12 +1155,13 @@ public function isDestructor(): bool {
 }
 
 class ReturnInfo {
-    const REFCOUNT_0 = "0";
+    // REFCOUNT_SCALAR (formerly REFCOUNT_0) is automatically applied for
+    // scalars, and not allowed for non-scalars
+    const REFCOUNT_SCALAR = "0";
```

Comment:
I don't like this renaming, because `refcount 0` is what PHP has historically been using: https://github.com/php/php-src/blob/eb1cdb5b721a849a2762a1136a28ff2340c1e103/Zend/Optimizer/zend_func_info.c#L41

Reply:
Sure, done